registry.py

```diff
@@ -1,8 +1,3 @@
-# NOTE: Must be before we import or call anything that may be synchronous.
-from gevent import monkey
-
-monkey.patch_all()
```

Comment:
@kleesc Assuming I understand correctly -- If this is removed, and Quay is ran without gunicorn (e.g. `python -m application.py`) then the monkey patching won't occur. Is this expected behaviour? Is there any downfall to keeping these monkey-patch lines in addition to adding them before the `gunicorn` workers?

Reply:
It's still being monkey patched in application.py.

Reply:
@kleesc Ahh, so you're basically identifying the entry-points and monkey patching them there? Smart!